test/e2e/extensions/drawer.cy.js

```diff
@@ -0,0 +1,63 @@
+import Course from '../../../src/course/en/course.json'
```

Comment:
Interesting... very interesting...

The tests can be run against a specified build output folder, which wouldn't have a `src/course` but an `{outputdir}/{coursedir}/` instead. So we're going to need some way of grabbing that stuff in the test runtime such that the json can be loaded from the correct place.

Reply:
A few people have mentioned they'd like to perform tests based upon the course content, effectively removing some Quality Assurance tasks. We should start from facilitating that point of view, even if we never get there. As the first test, this already relies on the defined course data.

Reply:
The resource specific e2e tests should live in resources.